js/event.js

```diff
@@ -1386,10 +1346,12 @@ export const toggleCurrentPage = async (route) => {
     const IDs = ['userDashboard', 'Notifications', 'userAgreements', 'userSettings', 'connectSamples', 'connectSupport', 'connectPayment'];
     IDs.forEach(id => {
         const element = document.getElementById(id);
-        if(element) {
-            element.addEventListener('click', () => {
-                removeActiveClass('navbar-nav', 'current-page');
-                element.parentNode.parentNode.classList.add('current-page');
+        if (element) {
+            const clonedElement = element.cloneNode(true);
+            element.parentNode.replaceChild(clonedElement, element);
```

Comment:
I read this is one way to handle event listener management (the old is removed and the new is created), which could be useful since toggle current page may be called frequently